examples/examples.test.ts

```diff
@@ -43,23 +43,28 @@ function validateVega(spec: TopLevelExtendedSpec) {
   assert(valid, errors && errors.map((err: Ajv.ErrorObject) => err.message).join(', '));
 }
 
+const suffixLength = '_future.vl.json'.length;
 describe('Examples', function() {
   const examples = fs.readdirSync('examples/specs');
 
   examples.forEach(function(example: string) {
-    if (
-      path.extname(example) !== '.json' ||
-      // Do not validate overlay example until we have redesigned it
-      example.indexOf('overlay') >= 0 ||
-      // Also ignore box-plot examples until we support selections
-      example.indexOf('box-plot') >= 0
-      ) {
+    if (path.extname(example) !== '.json') {
       return;
     }
     const jsonSpec = JSON.parse(fs.readFileSync('examples/specs/' + example));
 
     describe(example, function() {
       it('should be valid vega-lite with proper $schema', function() {
+        if (
+          // Do not validate overlay example until we have redesigned it
+          example.indexOf('overlay') >= 0 ||
+          // Also ignore box-plot examples until we support selections
+          example.indexOf('box-plot') >= 0 ||
+          // Also ignore all examples with "_future" suffix
+          example.lastIndexOf('_future.vl.json') === example.length - suffixLength
```

Comment:
This is an interesting way to make this test. An alternative might be a regular expression to verify that the string ends with the given suffix.